web-components/package.json

```diff
@@ -28,7 +28,8 @@
     "test": "vitest run --reporter=verbose --reporter=junit --coverage",
     "test:watch": "vitest",
     "design-tokens:generate": "node ./tokens/style-dictionary.cjs",
-    "icons:generate": "node src/components/icon/generate-icon-types.js"
+    "icons:generate": "node src/components/icon/generate-icon-types.js",
+    "chromatic": "VITE_IS_CHROMATIC=true chromatic"
```

Comment:
turns out I forgot to add this script in the last PR that actually makes sure, chromatic has the proper ENV var passed during build